### PR TITLE
Fixed the Hover on selected person in Finder

### PR DESCRIPTION
**Pull request by nileshprasad137**
Initially , there was no hover and cursor on selected item , now it has it.

screenshot 97

### CODE REVIEW

**Comment by teryhill** [comment]
Don't forget there are other style sheets 😎

**Comment by nileshprasad137** [comment]
oh yes! I'll fix those too. :smile:

**Comment by aethelwulffe** [comment]
Get those stylesheets, and let's move on this.  Much better usability in my opinion

**Comment by aethelwulffe** [comment]
Commit history is a little messy.  Tell me if it is too big of a pain to pull the meat of this into a new branch or not, then I will approve one way or another.  I don't want to be too picky.

**Comment by nileshprasad137** [comment]
Yes , I admit. Its horrible!!. I had updated the Docs on the master branch and so when I created a new branch on my repositiry, I had to retrace where the master branch of this organization lies. 
I had not worked with different branches before ,so completely messed it up. Sorry for that!

**Comment by nileshprasad137** [comment]
Still learning about different branches in Git.

**Comment by aethelwulffe** [comment]
No sweat man!  You are doing great in any case.  My usual cleanup is to just dump my changed files on top of a new branch (you can do that easy with GitHub for Windows anyway).  It involves dumping a pull request and starting over, but after closing, I just link the old to the new for the conversational bits.  90% of the code you will be committing will be after this point, as you only have the prototype in one CSS.  Of course, you could just add a new file and use a single line URL link...but that may be as much code as this change has!

File: interface/themes/style_light.css [hunk 1]
```diff
@@ -141,6 +141,13 @@ a:hover {
   color: #000000;
 }
 
+.sorting_1:hover
+{
+  color: #ff5555;
+  cursor: pointer;
+}
+
+
```
Comment (teryhill) [inline interface/themes/style_light.css]: looks good and tests with no issues

File: library/options.inc.php [hunk 2]
```diff
@@ -373,7 +373,7 @@ function generate_form_field($frow, $currvalue) {
       " value='" . $formDate . "' $disabled />";
     if (!$disabled) {
        $selector = "#form_{$field_id}";
-       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false, format:'" . $DateFormat . "'});";
+       $date_init .= "$('" . $selector . "').datetimepicker({timepicker: false,  format:'" . $DateFormat . "'});";
```
Comment (teryhill) [inline library/options.inc.php]: Why did you remove the fix?

Comment (nileshprasad137) [inline library/options.inc.php]: I fixed these in Updating Docs pull request, so I thought,fixing it in this pull request may create problems in future. 
Is it not the case?

Comment (aethelwulffe) [inline library/options.inc.php]: Yes, that is indeed the case.
I would like to get three other pull requests merged before this one just based on conflicts.  Last night a did a test merge of all the pull requests, and there is a specific series that can be merged without causing large issues.  We are developing a back-log at the moment, and we need to get things organized a bit.
Good call.

Comment (aethelwulffe) [inline library/options.inc.php]: Actually, this fix may not cause issues here.  I think you are right to leave it out though.  We need to merge your other commit first.

**Comment by aethelwulffe** [comment]
Hint:  To move development along, review other people's pull requests.  We need 2 upvotes (and no downvotes on a pull request, and then someone other than the originator should merge the commits.

**Comment by nileshprasad137** [comment]
Sure @aethelwulffe , I was a bit busy today , I'll see.

**Comment by aethelwulffe** [comment]
Hey, no pressure, but it is nice having two new really competent and active folks on-board, and since either of you are more competent than any of us with JS and CSS (obviously) having your eyes on things would be very helpful.

**Comment by aethelwulffe** [comment]
Nilesh,
Would you pretty please lay this file onto a new branch off of master and make a new pull request?  We have 17 commits with merges on one file change, and the commit history is a bit messy.  I would like to merge this immediately after you do so.  It has been checked out and agree upon.

**Comment by nileshprasad137** [comment]
@aethelwulffe Should I do it now? Or after fixing the hover on all the themes ?

**Comment by aethelwulffe** [comment]
Hey, your choice there, but you are right, it may be better to do it all at once.

**Comment by nileshprasad137** [comment]
I'll do it soon with all the modifications !

**Comment by nileshprasad137** [comment]
New pull request created https://github.com/LibreHealthIO/LibreEHR/pull/475

**Comment by aethelwulffe** [comment]
We have a new pull, so closing this request-> #475
